content/blog/2023/when-to-choose-a-responsive-web-app.md

```diff
@@ -0,0 +1,277 @@
+---
+title: When to Choose a Responsive Web App
+sub: Part 1 -- Responsive Web App vs Native Mobile App vs Progressive Web App
+author: sondra
+date: 2023-05-05
+tags:
+  - Article
+  - Business Development
+  - Push Notifications
+  - Recommendations
+  - Responsive Web Apps
+  - Startups
+  - User Experience
+image:
+  src: blog/2023/part1-web.jpg
+  alt: spider web with dew drops
+summary: |
+  The decision of what platform
+  to use to build your app
+  is quite important --
+  affecting project scope, timeline, and budget.
+  But understanding the differences between
+  a responsive web app,
+  a native mobile app,
+  and a progressive web app --
+  and deciding which one is right for your project --
+  can seem insurmountably difficult.
+  Welcome to part 1 of a three-part series
+  unpacking the reasons to choose
+  one platform over another.
+  Let's start with responsive web apps.
+---
+{% import 'embed.macros.njk' as embed %}
+
+You have an idea for a custom digital product you'd like to build, and it's time
+to decide how to build it. One of your friends swears you won't succeed unless
+you build a native mobile app for iPhone. A colleague argues that a responsive
+web app is the only way to go. Who's right? Are there alternative options?
+
+The great news is that you don't need to spend years becoming a web developer
+yourself in order to make a wise choice. In the following three-part series,
+we'll dig into the three main options for building mobile apps -- responsive web
+apps, native mobile apps, and progressive web apps -- and how to determine the
+best option for your project. As you read, think about who your app will serve,
+what their most pressing needs are, and how your app will meet those needs.
+
+<!-- **Read all three parts to compare app types:**
+
+1. [When to Choose a Responsive Web App](/2023/05/05/when-to-choose-a-responsive-web-app/)
+2. [When to Choose a Native Mobile App](/2023/06/07/when-to-choose-a-native-mobile-app/)
+3. [When to Choose a Progressive Web App](/2023/07/05/when-to-choose-a-progressive-web-app/) -->
+
+## What are Responsive Web Apps, Native Mobile Apps, and Progressive Web Apps?
+
+Here are the biggest differences in a nutshell.
+
+- **Responsive Web App (RWA)** -- An application that runs in web browsers, such
+  as Chrome, Firefox, or Safari, and responds to fit any screen size on any
+  device
+- **Native Mobile App** -- An app built specifically for one particular mobile
+  operating system, like iOS or Android, and downloaded from an app store
+- **Progressive Web App (PWA)** -- A responsive web app with additional
+  capabilities that mimic a native mobile app, such as push notifications and
+  offline mode
+
+{{ embed.img(
+  src='blog/2023/app-comparison-chart.jpg',
+  alt='A comparison chart showing the differences between
+    responsive web apps, progressive web apps, and native
+    mobile apps [summary](/rwa-pwa-native-comparison-chart-summary/)'
```

Comment:
It won't work to embed a link like this in `alt` text, so we'll need to find another way to surface this.